pkg/rescheduler/strategies/duplicates.go

```diff
@@ -21,24 +21,35 @@ import (
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api/v1"
+	//TODO: Change to client-go instead of generated clientset.
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 
 	"github.com/aveshagarwal/rescheduler/pkg/api"
 	"github.com/aveshagarwal/rescheduler/pkg/rescheduler/evictions"
 	podutil "github.com/aveshagarwal/rescheduler/pkg/rescheduler/pod"
 )
 
+// This strategy evicts all duplicate pods on node. The a pod is said to be a duplicate of other if
+// both of them have same name, kind and are within the same namespace. As of now, this strategy won't
+// evict daemonsets, mirror pods and pods with local storages.
```

Comment:
also need to include critical pods.

Reply:
Do, we include critical pods? I see that the check is not there in the FindDuplicatePods function.

Reply:
Somehow I missed that, now I have added a check for critical pods too.

Reply:
Its fine @aveshagarwal, I will include it.